front/app/utils/configs/participationMethodConfig/index.tsx

```diff
@@ -92,9 +90,6 @@ export type ParticipationMethodConfig = {
   showInputCount: boolean;
   hideAuthorOnIdeas?: boolean; // Hides the author on the idea pages/cards
   showIdeaFilters?: boolean; // Shows filters on the idea list
-  useProjectClosedCTABarStyle?: (
```

Comment:
This is not used anymore since we don't have continuous projects. If we have no active phase, participationMethod in ProjectCTABar is undefined, so the CTA bar won't even render.